sanic/mixins/routes.py

```diff
@@ -65,6 +74,7 @@ def route(
         static: bool = False,
         version_prefix: str = "/v",
         error_format: Optional[str] = None,
+        **kwargs,
```

Comment:
Perhaps also name this `ctx_kwargs` if no other kwargs are accepted. IDEs show the name while typing in code.